src/geff/spatial_graph/io.py

```diff
@@ -1,10 +1,12 @@
 from pathlib import Path
 from typing import Optional
 
+import numpy as np
 import spatial_graph as sg
 import zarr
 
 import geff
+import geff.utils
```

Comment:
I think one of these imports is redundant. I saw `geff.utils.validate` in at least one place.

Reply:
Since this code is gone now I will not update this (other than as needed for merging with main)

Reply:
(but also if you only `import geff`, I thought you couldn't then `geff.utils.validate(...)`)